### PR TITLE
WIP: [11.0][MIG] Migrate account_anglo_saxon_stock_move modules

### PR DESCRIPTION
**Pull request by keylor2906**

### CODE REVIEW

**Comment by keylor2906** [comment]
@dsabrinarg Could you test this please, the PR is WIP untill is sure that everything is fine.

Lodi dev instance: http://dev.lodigroup.com:30013/

Quenec dev instance: http://dev.lodigroup.com:30011/

File: account_anglo_saxon_stock_move_purchase/view/purchase_view.xml [hunk 1]
```diff
@@ -1,21 +1,55 @@
...
+                    string="Reconcile Accruals"
+                    attrs="{'invisible':[('to_be_reconciled','=',0)]}"
+                    class="oe_highlight">
+                    <field name="to_be_reconciled" invisible="True"/>
+                </button>
+            </xpath>
+        </field>
+    </record>
+
+    <!--<record id="view_purchase order stock accrual search" model="ir.ui.view">
```
Comment (hbto) [inline account_anglo_saxon_stock_move_purchase/view/purchase_view.xml:29]: There will be any way for the user to check:
- How many Purchase & Sales Order there are without fully reconciliation?
This is not that Urgent as the feature itself. Let us consult with the user to see if this feature
has really been used and schedule it for a next release.

Regards.

File: account_anglo_saxon_stock_move_sale/view/sale_view.xml [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
+                <button type="object" name="view_accrual" class="oe_stat_button"  context="{'search_default_group_by_product':True, 'search_default_unreconciled_n_ap_ar': True}" attrs="{'invisible':['|', ('state', 'in', ('draft','sent')), ('unreconciled_lines', '=', 0)]}" icon="fa-pencil-square-o">
+                    <field name="unreconciled_lines" string="Unreconciled Lines" widget="statinfo"/>
+                </button>
+
+                <button type="object" name="view_accrual" class="oe_stat_button"  context="{'search_default_group_by_product':True, 'search_default_group_by_reconcile':True}" attrs="{'invisible':[('state', 'in', ('draft','sent'))]}" icon="fa-pencil-square-o" string="See all Accruals"/>
+            </xpath>
+        </field>
+    </record>
+
+    <!--<record id="view_sale order stock accrual search" model="ir.ui.view">
```
Comment (hbto) [inline account_anglo_saxon_stock_move_sale/view/sale_view.xml:29]: Same case here.

As the case for purchase.

Regards.

**Review by hbto — APPROVED** [review]
In General This Looks Good To Me,

Please Merge at will.

Regards.

**Comment by sabrinagnu** [comment]
@josemoralesp merge please, functional tests 👍

**Pull request merged**